### PR TITLE
KERNEL: Update module version from 2.6.5 to 2.7

### PR DESCRIPTION
**Pull request by tvegas1**
Update kernel module version.

### CODE REVIEW

**Comment by brminich** [comment]
maybe NEWS file is worth updating in this PR as well?

**Comment by tvegas1** [comment]
Added NEWS file and started populating it.

File: NEWS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+## Current
```
Comment (brminich) [inline NEWS:7]: should we set a release version + date instead? Like we do in [UCX repo](https://github.com/openucx/ucx/releases)

Comment (tvegas1) [reply NEWS:7]: added specific version and date

**Review by brminich — DISMISSED** [review]

**Comment by brminich** [comment]
@yosefe, pls take a look

File: NEWS [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+### Features:
+### Bugfixes:
+
+## 2.7.0 (June 13, 2023)
+### Features:
+#### test
+ * Added gtest suite including port of existing tests
```
Comment (yosefe) [inline NEWS]: remove this section pls - not relevant for users

File: NEWS [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+#### kernel
```
Comment (yosefe) [inline NEWS]: `#### Kernel driver`

File: NEWS [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ * Added faulting of multiple pages at once, forward and backward
+ * Added compilation fixes for PPC on Ubuntu 18.04
```
Comment (yosefe) [inline NEWS:17]: move to bugfix section

File: NEWS [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ * Added vma_iterator and provided implementation for kernel < 6.2
```
Comment (yosefe) [inline NEWS]: Move to bugfixes as "Fixed vma_iterator compilation issue on kernel ..."

File: NEWS (hunk 2 above)
Comment (tvegas1) [reply NEWS]: fixed

File: NEWS (hunk 3 above)
Comment (tvegas1) [reply NEWS]: fixed

File: NEWS (hunk 4 above)
Comment (tvegas1) [reply NEWS:17]: fixed

File: NEWS (hunk 5 above)
Comment (tvegas1) [reply NEWS]: fixed

**Review by yosefe — APPROVED** [review]
pls squash

**Pull request merged**